cerk_port_mqtt_mosquitto/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "cerk_port_mqtt_mosquitto"
+version = "0.1.0"
+authors = ["Linus Basig <linus@basig.me>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = "0.4.0"
+env_logger = "0.7.1"
+url = "2"
+cerk = { version = "0.2", path = "../cerk" }
+cloudevents-sdk = "0.3.0"
+serde_json = "1.0"
+mosquitto-client = { git = "https://github.com/jsloth/mosquitto-client", rev="7913a98bfe6d35aaec9b541e7be4b817c42d91e0" } #fork of "0.1.5"
```

Comment:
we have to fix this...

Reply:
how would you like to fix it?

Reply:
https://github.com/ce-rust/cerk/issues/88